dev/airflow-pr

```diff
@@ -178,34 +189,43 @@ def merge_pr(pr_num, target_ref, title, body, pr_repo_desc, local):
         continue_maybe(msg)
         had_conflicts = True
 
+    pr_commits = get_json("{}/pulls/{}/commits".format(GITHUB_API_BASE, pr_num))
+
     merge_message_flags = []
 
     if squash:
         commit_authors = run_cmd(['git', 'log', 'HEAD..%s' % pr_branch_name,
                                  '--pretty=format:%an <%ae>']).split("\n")
-        distinct_authors = sorted(set(commit_authors),
-                                  key=lambda x: commit_authors.count(x), reverse=True)
-        primary_author = raw_input(
-            "Enter primary author in the format of \"name <email>\" (or press enter to use %s): " %
-            distinct_authors[0])
+        distinct_authors = sorted(
+            set(commit_authors),
+            key=lambda x: commit_authors.count(x),
+            reverse=True)
+        primary_author = click.prompt(
+            click.style(
+                'Enter the primary author in the format of \"name <email>\"',
```

Comment:
`click.prompt` displays the default in brackets after the prompt, so the new prompt looks like this:
```
Enter the primary author in the format of "name <email>" [jlowin <jlowin@apache.org>]: 
```